pkg/cloudprovider/provider/anexia/provider.go

```diff
@@ -119,14 +121,14 @@ func (p *provider) Create(ctx context.Context, machine *clusterv1alpha1.Machine,
 	}()
 
 	// provision machine
-	err = provisionVM(ctx, client)
+	err = provisionVM(ctx, a, client)
 	if err != nil {
 		return nil, anexiaErrorToTerminalError(err, "failed waiting for vm provisioning")
 	}
 	return p.Get(ctx, machine, data)
 }
 
-func provisionVM(ctx context.Context, client anxclient.Client) error {
+func provisionVM(ctx context.Context, a api.API, client anxclient.Client) error {
```

Comment:
why do we need this new argument, it's not used?

Reply:
Good catch. Looks like a leftover from refactoring.